### PR TITLE
add hidden-print to unneeded sections

### LINKED ISSUE #421: Print version
Would it be possible to get a better print.css? Currently the print version includes extraneous sections (comments/buttons/footers).

### PR DESCRIPTION
**Pull request by HyShai**
This will resolve #421

### CODE REVIEW

**Comment by matiasbeckerle** [comment]
Shouldn't be there any CSS media print styling for this to work accordingly? Couldn't find anything defined in the repository about `hidden-print`.

**Comment by ploeh** [comment]
@matiasbeckerle AFAICT, `hidden-print` is alread defined in `css/style.css`...

**Comment by matiasbeckerle** [comment]
Oh my bad, thanks. I've found it in the included bootstrap `3.1.1`. Yep -- this LGTM. Not a CSS expert but this was solved using an standard practice. I do agree too about the copyright notice.

File: _includes/themes/ploeh/default.html [hunk 1]
```diff
@@ -79,7 +79,7 @@
         {{ content }}
       </div>
 
-      <footer>
+      <footer class="hidden-print">
```
[the remark this answers is not in the document]
Comment (HyShai) [reply _includes/themes/ploeh/default.html]: Can we hide the "_with help from..._" part and just leave &copy; Mark Seeman {{date}}

[the remark this answers is not in the document]
Comment (ploeh) [reply _includes/themes/ploeh/default.html]: That would be fine, I think.

[the remark this answers is not in the document]
Comment (HyShai) [reply _includes/themes/ploeh/default.html]: Ok I made these changes

File: _includes/themes/ploeh/post.html [hunk 2]
```diff
@@ -25,7 +25,7 @@ <h1>{{ page.title }} <small>by Mark Seemann</small></h1>
     {% include JB/comments %}
   </div>
   
-  <div class="col-lg-4 col-md-4">
+  <div class="col-lg-4 col-md-4 hidden-print">
```
Comment (ploeh) [inline _includes/themes/ploeh/post.html]: I'm sorry I didn't see this the first time around, but I don't think the print view should hide the publication date. This is important information that a reader might need.

Comment (HyShai) [reply _includes/themes/ploeh/post.html]: I added an additional "Published Date" `div` which will show only when printing. (The current Published Date is inside the sidebar.)

**Comment by ploeh** [comment]
Thank you 👍

**Pull request merged**